src/server/visual-diff-reporter.js

```diff
@@ -51,10 +53,12 @@ function createData(rootDir, sessions) {
 		});
 	});
 
-	metadata.browsers = Array.from(browsers.values()).map(b => {
-		return { name: b.name, version: b.version };
-	});
-	writeFileSync(metadataPath, JSON.stringify(metadata, undefined, '\t'));
+	if (isCI || updateGoldens) {
```

Comment:
I assume we're ok with the browser versions being slightly off if you're using `--filter or --grep` with `--golden`, meaning any old goldens you don't update could've been created by an older browser. There isn't really a way around it without storing the browser info with every png like we'd originally thought about.  And it's only an issue locally, not in CI

Reply:
Good point, but yeah not worried about it. This stuff is really only useful in CI anyway -- if you're reusing old goldens locally from an old browser... I'm not sure what you're doing. I've almost talked myself into only using this information in CI, period, haha.